common/lib/protocol-definitions/src/protocol.ts

```diff
@@ -19,6 +22,9 @@ export enum MessageType {
     // Message used to reject a pending proposal
     Reject = "reject",
 
+    // Message send by client accepting proposal
+    Accept = "accept",
```

Comment:
Why do you think it's breaking? The new op does not have any new semantics (yet, they may show up later if service would want to coalesce them in the future). At the moment, this, as noop before, only moves MSN forward, and has no other semantics attached to it.

Reply:
From protocol change perspective, it does not matter where we change it - change needs to be done in a way that is safe for old clients, and I believe it's totally safe. The only place where it might be breaking is if someone has a switch over all MessageType's and now code will not compile because there is an extra case to deal with (op types is not supposed to be exclusive, so runtime code should always assume there are more types that are not known to client).
I've looked at Bohemia code, and there are only 2 places where MessageType is referenced - only to test for "op" type.
I'm willing to take that risk to speed up deployment - I have more changes on top of it (and final changes will need staging time).
Let me know what you think,

Reply:
I don't have strong feelings, it's a matter of the 'letter of the law' vs the 'spirit of the law'. If the change is safe then we should do it in `main`. It still needs the package.json changes to pass the build though..

Reply:
BTW, I can move only addition of MessageType.Accept to next :) The rest can go anywhere

Reply:
I've moved "breaking" changes to PR #11799 (against next), keeping the rest here.